docs/commands/intend.md

```diff
@@ -28,29 +28,29 @@ The intend command will return the full request and response of the interaction
 
 By default, the system will:
 
-* Use the Intent Model and Sample Utterances in the speechAssets folder under the current working directory
+* Use the Interaction Model in the models folder under the current working directory
+* If there's no Interaction Model, use the Intent Model and Sample Utterances in the speechAssets folder under the current working directory
```

Comment:
"If there's no Interaction Model, use" -> If there's no Interaction Model, it will use"